SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestPropertyReplication/SpatialTestPropertyReplication.cpp

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SpatialTestPropertyReplication.h"
+#include "Kismet/GameplayStatics.h"
+#include "ReplicatedTestActor.h"
+
+/**
+ * This tests that an Actor can replicate a property across the network during play.
+ * This test contains 1 Server and 2 Client workers.
+ *
+ * The flow is as follows:
+ * - Setup:
+ *  - The Server spawns one ReplicatedTestActor.
+ *  - Test:
+ *  - Both Clients check that they can see exactly 1 ReplicatedTestActor.
+ *  - The Server changes the ReplicatedProperty of the ReplicatedTestActor from "0" to "99".
+ *  - Both Clients check that the ReplicatedProperty is now set to "99".
+ * - Clean-up:
+ *  - ReplicatedTestActor is destroyed using the RegisterAutoDestroyActor helper function.
+ */
+
+ASpatialTestPropertyReplication::ASpatialTestPropertyReplication()
+	: Super()
+{
+	Author = "Ollie Balaam";
```

Comment:
Wanted to get your opinion on this as well.
Author field yay or nay?

Reply:
Is this question aimed at me? If so I'm pro because I can easily identity who wrote the test and owns maintaining it when it falls over.

Reply:
I also am in favour of this but would suggest changing this/including to the owner's email instead
e.g. `Author = "Thomas Yung (thomasyung@improbable.io)"`